lib/src/builders/method.dart

```diff
@@ -347,7 +395,27 @@ class _MethodBuilderImpl extends Object
         _property = property;
 
   @override
-  AstNode buildAst([Scope scope]) => buildFunction(scope);
+  AstNode buildAst([Scope scope]) {
+    if (_name != null) {
```

Comment:
Honestly just because dartfmt doesn't do a great job with > 80 char lambdas.

I can change it in a future PR